doc/source/user_guide/reshaping.rst

```diff
@@ -719,6 +719,58 @@ To choose another dtype, use the ``dtype`` argument:
     pd.get_dummies(df, dtype=bool).dtypes
 
 
+To convert a "dummy" or "indicator" ``DataFrame``, into a categorical ``DataFrame``
```

Comment:
add a versionadded 1.4.0 tag

Reply:
i find the parens a bit weird here, can you either say (or from a categorical 'Series' or drop them)

Reply:
Removed all the `Series` (and `to_series`) related parts